include/ck/tensor_operation/gpu/grid/gridwise_gemm_pipeline_v1.hpp

```diff
@@ -148,7 +148,9 @@ struct GridwiseGemmPipeline_v1<AGridDesc,
 
                 blockwise_gemm.Run(a_block_buf, b_block_buf, c_thread_buf);
 
+#if !CK_EXPERIMENTAL_INTER_WAVE_SCHEDULING
```

Comment:
I think it's better to have a new gridwise gemm pipeline for this improvement

Reply:
Addressed in 56790a62